Completing some isssues + fix in readme

SlejmUr opened the pull request:
Fix null in contributor.md
Add more classes on Location
Add Survey models
Add Exceptions for AbstactCoreHttpController
Add basic CryptoService
Add simple versioncheck.
Add GetEftVersion method in EftHttpContext

SlejmUr committed:
Fix null in contributor.md
Add more classes on Location
Add Survey models
Add Exceptions for AbstactCoreHttpController
Add basic CryptoService
Add simple versioncheck.
Add GetEftVersion method in EftHttpContext

SlejmUr commented:
Should complete these:
287

Partially complete these:
290 (Require some Items)
141 & 135 (Need check for it actually works, plus implementation)

SlejmUr committed:
Fix RequestBodyNotParsable error
Implement CryptographyService inside the CoreHttpContext
CryptographyService is now static. (Should we go back to Instance version or its good like this?)

SlejmUr committed:
Fix Test HttpClient to include version.

seionmoya reviewed (changes requested):
Thank you for your PR! Good stuff in there.

I should've mentioned in the CONTRIBUTING docs that abbr. names for `Stream` derivative types are fine. Sorry for that!

seionmoya commented:
Resolves:
#287 

Partially resolves:
#135 
#141 
#290

SlejmUr commented on Fuyu.Backend.EFT/Controllers/Http/CheckVersionController.cs:
```diff
@@ -13,17 +13,26 @@ public CheckVersionController() : base("/client/checkVersion")
-            // --seionmoya, 2024-11-18
+            // TODO: Add global constant somewhere where we can define the supported version of EFT/Arena?
+            // -- slejmur, 2025-01-09
+            string currentVersion = "0.16.0.2.34510";
```
Will do, but as comment says where should we have this as constant?
If we support eft, arena we need some constants somewhere where we declare globally what version we support

SlejmUr committed:
update all csproj to use LangLevel 13. (net9.0)
Add AesHelper from CryptographyService
Function from CryptographyService to AesHelper.

SlejmUr committed:
namespace_declarations on file_scoped is producing warnings rather than suggestion
+ format code.

SlejmUr committed:
Merge branch 'dev' into dev

SlejmUr commented:
Resolved conflicts too

SlejmUr committed:
Fix the resolve
